crane/container.go

```diff
@@ -206,8 +206,19 @@ func (r *RunParameters) Entrypoint() string {
 
 func (r *RunParameters) Env() []string {
 	var env []string
-	for _, rawEnv := range r.RawEnv {
-		env = append(env, os.ExpandEnv(rawEnv))
+	if r.RawEnv != nil {
+		switch rawEnv := r.RawEnv.(type) {
+  	  case []interface{}:
+  			for _, v := range rawEnv {
+  				env = append(env, os.ExpandEnv(v.(string)))
+  			}
+      case map[interface{}]interface{}:
+  			for k, v := range rawEnv {
+          env = append(env, os.ExpandEnv(k.(string)) + "=" + os.ExpandEnv(v.(string)))
+  			}
+  		default:
+  			print.Errorf("env is of unknown type!")
```

Comment:
that should be a fatal error (panic) and the faulty value should be passed along